docs/python_airflow_operator.md

```diff
@@ -81,6 +81,22 @@ Runs an Armada job and calls the job_service_client for polling.
 
 
 
```

Comment:
These docs don’t match the code you have here.

Reply:
yah I am updating them!!